docs/content/en/docs/tasks/cluster/cluster-upgrades/baremetal-upgrades.md

```diff
@@ -16,12 +16,24 @@ The upgrade command also upgrades core components of EKS Anywhere and lets the u
 >**_NOTE:_** Currently only Minor Version Upgrades are support for Bare Metal clusters. No other aspects of the cluster upgrades are currently supported.
 >
 
+### Modular Cluster Upgrades
+
+To allow users to be explicit when deciding which clusters to upgrade, workload clusters are not affected by management cluster upgrades.
+Therefore, each cluster must be individually upgraded to enjoy the latest features on all clusters.
+
+When triggering a workload cluster upgrade after upgrading the management cluster, it will not only reflect the changes in the cluster spec, but also any changes derived from the difference in business logic between the old and new EKS-A cluster controllers.
+This difference in logic can trigger a machine rollout, even if the updates to the cluster spec didn't require one (for example, scaling down a worker node group).
+
 ### EKS Anywhere Version Upgrades
 
 We encourage that you stay up to date with the latest EKS Anywhere version, as new features, bug fixes, or security patches might be added in each release. You can find the content, such as supported OS versions and changelog, of each EKS Anywhere version on the [What’s New]({{< relref "../../../reference/changelog/" >}}) page.
 
 Download the [latest or target EKS Anywhere release](https://github.com/aws/eks-anywhere/releases/) and run `eksctl anywhere upgrade cluster` command to upgrade a cluster to a specific EKS Anywhere version.
 
+We provide a maximum skew version support of one EKS Anywhere minor version for management and workload clusters.
```

Comment:
We need to add a comment on what happens if a workload cluster creation starts on v0.15.x, then the customer upgrades management cluster to v0.16. Does it work? Does it break?